recipe/meta.yaml

```diff
@@ -37,9 +38,9 @@ requirements:
     - python
     - pip
     - setuptools
-    - llvmlite 0.36.*
-    - numpy
-    - tbb-devel >=2019.5,<=2020.3
+    - llvmlite 0.37.*
+    - numpy >=1.17,<1.21
```

Comment:
Would just add this constraint to `run`. Maybe as a separate line

Reply:
Seems to have helped, though I don't really understand why this didn't work from [reading the docs](https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy).